addon/services/file-manager.js

```diff
@@ -141,25 +244,65 @@ export default Ember.Service.extend({
                 headers: {
                     Authorization: `Bearer ${accessToken}`
```

Comment:
Good point -- this should definitely be configurable, I'll hook it up to the existing setup